network/src/commonMain/kotlin/com/wire/kalium/network/api/message/MessageApiImp.kt

```diff
@@ -36,13 +36,13 @@ class MessageApiImp(private val httpClient: HttpClient) : MessageApi {
         parameters: MessageApi.Parameters.DefaultParameters,
         conversationId: String,
         option: MessageApi.MessageOption
-    ): KaliumHttpResult<SendMessageResponse> {
+    ): NetworkResponse<SendMessageResponse> {
 
         suspend fun performRequest(
```

Comment:
performRequest here can throw ktor Exceptions

Reply:
But I thought we wanted to "swallow" these exceptions? Or at least map them to `KaliumExceptions` like objects, so that Ktor implementation details could be hidden to anyone that will consume Kalium

Reply:
I changed it to throw a KaliumException instead